src/app/components/Footer/Footer.js

```diff
@@ -4,8 +4,10 @@ import Text from '../Text';
 import { Section, Icon, Space } from '../../Symbols.js';
 import socials from '../../../assets/data/socials.json';
 
+import cn from './Footer.module.scss';
+
 const Footer = () => (
-	<Section id='bot' className='short'>
+	<Section className={`short ${cn.footer}`}>
```

Comment:
I added clsx to the library list (similar to classNames, if you've heard of that) so you can do instead of:
```{`short ${cn.footer}`}```
rewrite as:
```{clsx(cn.footer, 'short')}```

but I think end goal is to just completely remove these global classnames. clsx is more for logic basic classnames for ex.
```{clsx(cn.card, showDetails && cn.expanded)}```